src/nix/meson.build

```
@@ -212,18 +212,23 @@ nix_symlinks = [
   'nix-store',
 ]
 
+name_suffix = ''
```

```suggestion
executable_suffix = ''
```

I'm surprised to see that meson doesn't provide this piece in, say, `host_machine`, so it is what it is.